### DISCUSSION TITLE
Does a strategy exist with mergerfs to keep backup mirrored on 3 different hard drives?

### DISCUSSION THREAD

**Discussion by bjce** [trapexit/mergerfs#1378] [General]
Thank you to the whole team developing/maintaining mergerfs which looks like a great piece of software. I hope the question has not been previously but I could not find the answer elsewhere.

Let's says I have a server and 3 different hard drives (e.g. HDD_A: 6TB, HDD_B: 6TB, HDD_C: 6TB) and the following requirements:

- I want that a new file is being automatically copied on the each of the 3 hard drives. 
- I want them to be individually usable, meaning, if I remove one (or all) of them from the server, I want to be able to read it/them out of the box straight ahead (without having to setup a software layer on top such as RAID).
- If I remove one of the 3 hard drives (to keep one of the backups off-site), I want the system still to be functioning and mirroring the backup-data on the 2 remaining hard drives.

I this configuration possible with mergerfs (e.g. with 2 parity drives?)

**Comment by trapexit** [comment]
mergerfs is not RAID and it's not a backup solution. So it doesn't do that collection of things by itself.

1. mergerfs has no such feature. You would need to use an external setup to accomplish that.
2. You should read about mergerfs policies. They are how mergerfs chooses a file/branch. If you have the same relative path for files across branches then mergerfs will chose one based on the policy.
3. A core feature/design decision of mergerfs is that things can be added and removed at will.  Read over the docs: https://github.com/trapexit/mergerfs?tab=readme-ov-file#can-mergerfs-be-used-with-filesystems-which-already-have-data--are-in-use

I would just use rsync or rclone or unison or similar to regularly sync files across the filesystems. Same as if mergerfs wasn't there.

**Reply by bjce** [reply]
dear @trapexit thank you very much for your very quick and detail answer, I will then use rsync
